Sources/PointFree/Account/Account.swift

```diff
@@ -842,15 +843,15 @@ let blockSelectClass =
     | Class.type.fontFamily.inherit
 
 private struct AccountData {
-  let currentUser: Database.User
-  let emailSettings: [Database.EmailSetting]
-  let episodeCredits: [Database.EpisodeCredit]
+  let currentUser: User
+  let emailSettings: [EmailSetting]
+  let episodeCredits: [EpisodeCredit]
   let stripeSubscription: Stripe.Subscription?
   let subscriberState: SubscriberState
-  let subscription: Database.Subscription?
-  let subscriptionOwner: Database.User?
-  let teamInvites: [Database.TeamInvite]
-  let teammates: [Database.User]
+  let subscription: Models.Subscription?
```

Comment:
we talked about this irl but just for posterity: we discussed changing `Stripe.Susbcription` to StripeSubscription` so that we could then remove these `Models` qualifications.